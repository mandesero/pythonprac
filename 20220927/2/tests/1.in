5,4,3,124,453,65,32,45
