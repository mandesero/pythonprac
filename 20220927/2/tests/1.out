[3, 453, 4, 32, 5, 65, 45, 124]
